### DISCUSSION TITLE
How to force modules to load even without sending any requests to the host?

### DISCUSSION THREAD

**Discussion by xperiandri** [OrchardCMS/OrchardCore#14418] [Q&A] [answered]
I have a processing app that has only health check endpoints.
However, when I debug it, I do not want to call any HTTP endpoint, but just have it started and all the enabled features automatically loaded.
How can I force such behavior?

**Comment by jtkech** [comment, marked as answer]
You can configure the `BackgroundServiceOptions.ShellWarmup` to true.

Can be done e.g. in appsettings under the `OrchardCore:OrchardCore_BackgroundService` section.

**Reply by xperiandri** [reply]
Works, thanks!
`.AddBackgroundService()` is required for Orchard Core Frramework
